target/consensource-ui
# Update yargs-parser & serialize-javascript packages in response to dependabot

## Proposed change/fix

Update yargs-parser & serialize-javascript packages in response to dependabot

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

`npm i`
`npm run build`
`npm run start:dev`
Check console for errors
Visit personas in browser
